bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/launch/LaunchUtils.java

```diff
@@ -142,4 +146,19 @@ private static String populateConfigDetails(List<Path> paths, Map<String, String
         }
         return null;
     }
+
+    public static TomlDetails getTestConfigPaths(Module module, String pkgName, String sourceRoot) {
+        String moduleName = module.getName();
+        Path testConfigPath = Paths.get(sourceRoot);
+        if (!moduleName.equals(pkgName)) {
+            testConfigPath = testConfigPath.resolve(MODULES_ROOT)
+                    .resolve(moduleName.substring(moduleName.indexOf(DOT) + 1));
+        }
+        testConfigPath = testConfigPath.resolve(TEST_DIR_NAME).resolve(CONFIG_FILE_NAME);
+        if (!Files.exists(testConfigPath)) {
+            return new TomlDetails(new Path[]{}, null);
+        } else {
+            return new TomlDetails(new Path[]{testConfigPath}, null);
```

Comment:
Why is this line not covered? I think, we have integration tests for this. Shall we verify?

Reply:
+1, Other changes LGTM.

Reply:
That line is covered at least by this test package https://github.com/ballerina-platform/ballerina-lang/tree/master/tests/jballerina-integration-test/src/test/resources/configurables/testErrorProject/configPkg. And there are some other configurable tests also covering this path. 
![Screenshot from 2023-05-16 09-06-14](https://github.com/ballerina-platform/ballerina-lang/assets/35595065/44b6dca4-3bf1-429e-9410-c62eed32c4b0)

Reply:
I'm not sure why codecov tool marks this line as not covered. Even it should be marked as partially covered if it's not checked for all the possible cases right?

Reply:
I think it's because we are not using the jar execution here. 
For integration tests, we can only pass the jacoco agent through jar execution. So I guess this is expected. 
Let's check this separately.